src/terraform/devcontainer-feature.json

```diff
@@ -43,7 +43,7 @@
             "settings": {
                 "terraform.languageServer": {
```

Comment:
Also bump the patch version here https://github.com/devcontainers/features/blob/5336b419313fd75b01da75766e5b1fc1ddefb2c9/src/terraform/devcontainer-feature.json#L3 ? We release features every time a PR is merged, these changes won't take effect unless there is a version change from what currently exists :)

Reply:
Thanks for catching! Just bumped to 1.0.4, let me know if it looks alright to you 😄

Reply:
Looks good! 👍